### PR TITLE
Fix ripples in buttons on Android

### PR DESCRIPTION
**Pull request by osdnk**
Fixes ripples in:
1. Fabs
2. Savings sheet
3. Unique token
4. Profile Avatar

Now they are no bigger than the button size

tested and no regressions on ios

### CODE REVIEW

**Review by brunobar79 — COMMENTED** [review]
@osdnk  this is a huge improvement! 🙏 

I tested all the components both on Android & iOS and I found two regressions on iOS:

- FAB shows white border when pressed (we might need to apply the styles to the container, at least the bg color) https://recordit.co/fZTXYMtQkB

- Profile Button on the wallet navbar is not centered anymore while animating: https://recordit.co/yQwxnxo2Nf

**Comment by christianbaroni** [comment]
> @osdnk this is a huge improvement! 🙏
> 
> I tested all the components both on Android & iOS and I found two regressions on iOS:
> 
> * FAB shows white border when pressed (we might need to apply the styles to the container, at least the bg color) https://recordit.co/fZTXYMtQkB
> * Profile Button on the wallet navbar is not centered anymore while animating: https://recordit.co/yQwxnxo2Nf

last one isn't a regression, all the header buttons scale down to the left or right. hints at which way it'll navigate

**Comment by osdnk** [comment]
> Profile Button on the wallet navbar is not centered anymore while animating: recordit.co/yQwxnxo2Nf 

it's okay!

File: src/components/profile/AvatarCircle.js [hunk 1]
```diff
@@ -55,10 +55,10 @@ export default function AvatarCircle({
 
   return (
     <ButtonPressAnimation
+      disabled={android}
```
[the remark this answers is not in the document]
Comment (osdnk) [reply src/components/profile/AvatarCircle.js:58]: it should be `isAvatarPickerAvailable && android`.
For Android, we have BPA deeper in a hierarchy

**Comment by osdnk** [comment]
Fixed all things, @brunobar79

**Review by brunobar79 — APPROVED** [review]
💯

**Pull request merged**